asp_plot/altimetry.py

```diff
@@ -16,12 +16,9 @@
 from asp_plot.stereopair_metadata_parser import StereopairMetadataParser
 from asp_plot.utils import ColorBar, Raster, glob_file, save_figure
 
-icesat2.init("slideruleearth.io", verbose=True)
```

Comment:
This shoudl be inside the Altimetry __init__, partially addresses https://github.com/uw-cryo/asp_plot/issues/71